test/functional/provider/python_spec.lua

```diff
@@ -15,49 +25,58 @@ do
   end
 end
 
-describe('python commands and functions', function()
-  before_each(function()
-    clear()
-    command('python import vim')
-  end)
+before_each(function()
```

Comment:
busted docs only show examples inside `describe()`. Just wanted to be sure.

Anyways, LGTM.